hubs/README.md

```diff
@@ -1,6 +1,11 @@
 # Hubs examples
 
-This directory holds different [hub](https://bump.sh/#hubs) examples
-(one hub per directory) which are deployed to
-[bump.sh](https://bump.sh) to quickly and easily create API catalogs
-from OpenAPI & AsyncAPI contract files.
+This directory holds OpenAPI & AsyncAPI Example API Contract files for API Portals or [Hubs](https://docs.bump.sh/help/hubs/) as we call them.
```

Comment:
The help page about Hubs isn't very good for now (it mostly deals with the CLI usage of Hubs) what about linking to our marketing page about Hubs?

```suggestion
This directory holds OpenAPI & AsyncAPI Example API Contract files for API Portals or [Hubs](https://bump.sh/api-catalog) as we call them.
```